Sources/AppStoreConnectCLI/Commands/TestFlight/Apps/ReadAppCommand.swift

```diff
@@ -0,0 +1,54 @@
+// Copyright 2020 Itty Bitty Apps Pty Ltd
+
+import AppStoreConnect_Swift_SDK
+import ArgumentParser
+import Foundation
+
+struct ReadAppCommand: CommonParsableCommand {
+
+    static var configuration = CommandConfiguration(
+        commandName: "read",
+        abstract: "Find and read app info"
+    )
+
+    @OptionGroup()
+    var common: CommonOptions
+
+    @Argument(
+        help: ArgumentHelp(
+            "The app AppStore ID. eg. 432156789 or app bundle identifier. eg. com.example.App",
+            discussion: "Please input either app id or bundle Id",
+            valueName: "app-id / bundle-id"
+        ),
+        transform: Identifier.init
+    ) var identifier: Identifier
+
+    enum Identifier {
+        case appId(String)
+        case bundleId(String)
+
+        init(_ argument: String) {
+            switch Int(argument) == nil {
+            case true:
+                self = .bundleId(argument)
+            case false:
+                self = .appId(argument)
+            }
+        }
+    }
+
+    func run() throws {
+        let service = try makeService()
+
+        var app: App
```

Comment:
Since all execution paths initialize this variable you should technically be able to use `let` here
```suggestion
        let app: App
```